UltiSnips/gitcommit.snippets

```diff
@@ -0,0 +1,95 @@
+snippet chore "create CHORE: commit message"
+CHORE: ${1:add build script}
```

Comment:
yeah it provide the base place holders for writing the commit messages mentioned in the previous blog posts